src/app/features/register-student/register-student.component.html

```diff
@@ -0,0 +1,83 @@
+<div class="container">
+    <div class="register-form-students">
+        <form class="form-students" [formGroup]="formGroup">
```

Comment:
change form name to be more precise, it can be `formRegisterStudent`

Reply:
In addition, don't be redundant with the form names, use only `name`, `lastName`, and so on